### ISSUE TITLE
Flaky Apihub GraalVM IT: Missing jar-file

### ISSUE THREAD

**Issue by burkedavison** [googleapis/google-cloud-java#11164]
https://fusion2.corp.google.com/invocations/bb99cb99-c4f8-4cac-92cf-c29641ef816e/targets/cloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fnightly%2Fgraalvm-sub-jobs%2Fnative-b%2Fgraalvm-native-14;config=default/log
```
08:05:41:069 [ERROR] Failed to execute goal org.graalvm.buildtools:native-maven-plugin:0.10.2:test (test-native) on project google-cloud-apihub: Missing jar-file for com.google.api.grpc:grpc-google-cloud-apihub-v1:jar:0.3.0-SNAPSHOT:test. Ensure that native-maven-plugin runs in package phase. -> [Help 1]
```

**Comment by burkedavison** [comment]
Replicated over several days:
https://fusion2.corp.google.com/ci/kokoro/prod:cloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fnightly%2Fgraalvm-native-b/activity/570751c9-c8a3-43b8-ab20-0042922fb22b/log

https://fusion2.corp.google.com/ci/kokoro/prod:cloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fnightly%2Fgraalvm-native-b/activity/23e0f035-7464-4c66-a09f-42b191d8a773/log

https://fusion2.corp.google.com/ci/kokoro/prod:cloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fnightly%2Fgraalvm-native-b/activity/8c0c6a50-c6e8-4b2b-a2e0-b2e9eaeb99b3/log

https://fusion2.corp.google.com/ci/kokoro/prod:cloud-devrel%2Fclient-libraries%2Fjava%2Fgoogle-cloud-java%2Fnightly%2Fgraalvm-native-b/activity/6d464195-a194-409e-8c30-8e7a8a308686/log

**Comment by JoeWang1127** [comment]
The job `prod:cloud-devrel/client-libraries/java/google-cloud-java/nightly/graalvm-sub-jobs/native-b/graalvm-native-14` has not failed for this error since Sep. 26, I think the error is fixed in #11185.
